### PR TITLE
Get program name from cache and display program certificate

### PR DESCRIPTION
**Pull request by muak50**
### Description

[LP-1937](https://philanthropyu.atlassian.net/browse/LP-1937)

Get program name from cache and display program certificate in My Certificates page.

### CODE REVIEW

File: openedx/features/student_certificates/helpers.py [hunk 1]
```diff
@@ -113,17 +118,23 @@ def get_credential_certificates(user):
...
-                'certificate_url': certificate_url,
-                'completion_date': completion_date.strftime(COMPLETION_DATE_FORMAT),
-                'is_program_cert': True,
-            })
+
+        program_uuid = credential['credential']['program_uuid']
+        program = cache.get(PROGRAM_CACHE_KEY_TPL.format(uuid=program_uuid))
+        if not program:
+            log.error('Program not found! Cache might be empty or don\'t contains program against uuid: %s',
+                      program_uuid)
```
Comment (zeshanarifios) [inline openedx/features/student_certificates/helpers.py:126]: please use format instead of %s

Comment (ahmed-faraz46) [reply openedx/features/student_certificates/helpers.py:126]: * `doesn\'t`

**Comment by ahmed-faraz46** [comment]
@muak50 let's please update commit message and add ticket number in it.

File: openedx/features/student_certificates/helpers.py (hunk 1 above)
Comment (muak50) [reply openedx/features/student_certificates/helpers.py]: Kindly specify a reason why `format` should be used here?

Comment (muak50) [reply openedx/features/student_certificates/helpers.py]: @ahmed-faraz46 Done

Comment (ahmed-faraz46) [reply openedx/features/student_certificates/helpers.py]: @muak50 I am not sure if we have set some convention or not, but at most of the places we are using format, and we should decide on something and document it in our code review document.

Comment (muak50) [reply openedx/features/student_certificates/helpers.py]: @ahmed-faraz46 We are using both conventions. If we are using multiple place-holders than '.format' seems to be more appropriate with placed parameters. Here we are dealing with just one place-holder string, so both can be used here.

Comment (zeshanarifios) [reply openedx/features/student_certificates/helpers.py]: @muak50 
%s is old style string formatting and .format is new style formating. And now in our new code we always prefer to use .format for formating string.

Comment (zeshanarifios) [reply openedx/features/student_certificates/helpers.py]: https://realpython.com/python-string-formatting/

Comment (muak50) [reply openedx/features/student_certificates/helpers.py]: @zeshanarifios I updated code based on your suggestion. Thanks

**Pull request merged**